client/src/Refactor/UploadForm.jsx

```diff
@@ -1,12 +1,17 @@
 import Dropzone from 'react-dropzone';
+import UploadForm from './UploadForm.css'
 import React, { Component } from 'react';
-var request = require('superagent')
-var apiBaseUrl = "http://localhost:3000/api/"
+import { Button, Row, Col, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
+var request = require('superagent');
+var apiBaseUrl = "http://localhost:3000/api/";
+
 
 class UploadScreen extends Component{
   constructor(props){
     super(props);
     this.state={
+      name: '',
+      price: '',
       filesToBeSent:[],
       filesPreview:[],
       printcount: 2
```

Comment:
linter's usually like a comma even on the last line of an object's key-value pairs.